src/CallControl/Action/MakeCallAction.ts

```diff
@@ -5,7 +5,7 @@ export class MakeCallAction extends Action {
         super('makeCall', {
             number: number,
             phone: phone,
-            auto_answer: autoAnswer,
+            auto_answer: autoAnswer ? Number(autoAnswer) : 0,
```

Comment:
This is invalid, not setting auto answer should use the system defined setting. Not disable auto answer.

Reply:
True, i changed it to -1 which means fetch it from the users' settings